rskj-core/src/main/java/org/ethereum/vm/program/Program.java

```diff
@@ -1086,6 +1086,10 @@ public ProgramResult getResult() {
         return result;
     }
 
+    public ProgramInvoke getInvoke() {
```

Comment:
this one is only needed for tests, isn't it? Can it be market with the visible for testing annotation? Also can it be package-level visibility?

Reply:
We will also be using it in TransactionExecutor to call the `init` for the `Environment` contract there too.